src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -187,7 +188,14 @@ exchange(Peer) ->
     case mochiglobal:get(delta_mode, false) of
         true ->
             %% Anti-entropy mechanism for causal consistency of delta-CRDT.
-            gen_server:call(?MODULE, {exchange, Peer}, infinity);
+            {ok, Pid, GC_Counter} = gen_server:call(?MODULE, {exchange, Peer}, infinity),
+            MaxGCCounter = mochiglobal:get(delta_mode_max_gc_counter, 7),
+            case GC_Counter == MaxGCCounter of
```

Comment:
Erlang, more stylistically, would have this variable be `GCCounter` and not `GC_Counter`; no underscores.